### PR TITLE
Add/pdms

### PR DESCRIPTION
**Pull request by fgrunewald**
wait for #292 to put info with link to parameters

### CODE REVIEW

**Comment by fgrunewald** [comment]
@ricalessandri can you do a small spell check on line 37 and 75ff of PDMS.martini3.ff

Otherwise this one should be good to go

**Comment by ricalessandri** [comment]
Ok. Will do it later today.

File: polyply/data/martini3/PDMS.martini3.ff [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+[ dihedrals ]
+;   i     j     k     l   funct     dihedral   force.c.   mult.
+  PDMS   +PDMS   ++PDMS   +++PDMS       1         1.18    1.4       1     ; 2
+;
+[ moleculetype ]
+PDMSter  1
+[ atoms ]
+  1    DMS      1     PDMSter     END   1      0.00000     72.00
+[ info ]
+Parameters for this model are custom and available for download here: https://github.com/marrink-lab/martini-forcefields/blob/main/martini_forcefields/regular/v3.0.0/gmx_files_contributed/DMS_martini3_v1.itp
```
Comment (ricalessandri) [inline polyply/data/martini3/PDMS.martini3.ff:37]:
```suggestion
Parameters for this model are custom and available for download at: https://github.com/marrink-lab/martini-forcefields/blob/main/martini_forcefields/regular/v3.0.0/gmx_files_contributed/DMS_martini3_v1.itp
```

File: polyply/data/martini3/PDMS.martini3.ff [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+[ link ]
+resname "PDMS|PDMSter"
+[ dihedrals ]
+PDMS  +PDMS  ++PDMS  +++END  1    1.85    1.2    1 {"group": "PDMS-END"}
+
+[ link ]
+[ atoms ]
+PDMS {"resname": "PDMS"}
+[ warning ]
+You should patch PDMS with a terminal residue. Use -seq PDMSter:1 PPE:4 PDMSter:1
```
Comment (ricalessandri) [inline polyply/data/martini3/PDMS.martini3.ff:75]:
```suggestion
You should patch PDMS with a terminal residue. Use -seq PDMSter:1 PDMS:4 PDMSter:1
```

File: polyply/data/martini3/PDMS.martini3.ff [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+[ non-edges ]
+PDMS +PDMS
+PDMS +END
+
+[ link ]
+[ atoms ]
+PDMS {"resname": "PDMS"}
+[ warning ]
+You should patch PDMS with a terminal residue. Use -seq PDMSter:1 PPE:4 PDMSter:1
```
Comment (ricalessandri) [inline polyply/data/martini3/PDMS.martini3.ff:84]:
```suggestion
You should patch PDMS with a terminal residue. Use -seq PDMSter:1 PDMS:4 PDMSter:1
```

File: polyply/data/martini3/PDMS.martini3.ff (hunk 2 above)
Comment (ricalessandri) [reply polyply/data/martini3/PDMS.martini3.ff:75]: Another thing is, "4" is a bit arbitrary there of course but I guess this should be clear enough

File: polyply/data/martini3/PDMS.martini3.ff (hunk 3 above)
Comment (ricalessandri) [reply polyply/data/martini3/PDMS.martini3.ff:84]: Another thing is, "4" is a bit arbitrary there of course but I guess this should be clear enough

File: LIBRARY.md [hunk 4]
```diff
@@ -25,4 +25,4 @@
 |Dextran                        |DEX                      |                                                                       |[martini3](polyply/data/martini3/dextran.martini3.ff) |
 |DNA nucleobases                |Dx, Tx5, Dx3 w/ x=T,G,A,C|[parmbsc1](polyply/data/parmbsc1/dna_final.ff)                         |[martini2](polyply/data/martini2/DNA_M2.ff)           |
 |Aminoacids                     |3 letter code            |                                                                       |[martini3](polyply/data/martini3/aminoacids.ff)       |
-
+|PDMS                           |DMS                      |                                                                       |[martini3](polyply/data/martini3/PDMS.martini3.ff)
```
Comment (ricalessandri) [inline LIBRARY.md:28]:
```suggestion
|polydimethylsiloxane                           |PDMS                      |                                                                       |[martini3](polyply/data/martini3/PDMS.martini3.ff)
```

**Review by ricalessandri — CHANGES_REQUESTED** [review]
Some small fixes, some are a bit subjective so feel free to accept/reject.
I also already commited a fix to LIBRARY.md

**Comment by fgrunewald** [comment]
@ricalessandri thanks for catching the typos and copy errors

**Review by ricalessandri — APPROVED** [review]
No problem!

**Pull request merged**